### PR TITLE
Included awscli & credentials installation details #2234

### PR DESCRIPTION
**Pull request by rmsmani**
**Please refer to the contribution guidelines in the README when submitting PRs.**
#2234

### CODE REVIEW

**Comment by rmsmani** [comment]
@whummer 
Please review

File: README.md [hunk 1]
```diff
@@ -109,10 +109,26 @@ The easiest way to install LocalStack is via `pip`:
 pip install localstack
 ```
 
+Installing the `awscli` for running the aws commands through `cli`
```
Comment (whummer) [inline README.md:112]: Please remove this section - we already have a section `Accessing the infrastructure via CLI or code` further below.

File: README.md [hunk 2]
```diff
@@ -109,10 +109,26 @@ The easiest way to install LocalStack is via `pip`:
...
+
+```
+pip install awscli
+```
+
 **Note**: Please do **not** use `sudo` or the `root` user - LocalStack
 should be installed and started entirely under a local non-root user. If you have problems
 with permissions in MacOS X Sierra, install with `pip install --user localstack`
 
+## Setting up local region and credentials to run localstack
```
Comment (whummer) [inline README.md:122]: Can we please move this paragraph into the `Accessing the infrastructure via CLI or code` section further below.

File: README.md [hunk 3]
```diff
@@ -296,6 +296,21 @@ local directory **ls_tmp** must contains the three files (server.test.pem, serve
 
 You can point your `aws` CLI to use the local infrastructure, for example:
 
+Use the below command to install `aws CLI`, if not installed already.
```
Comment (whummer) [inline README.md:299]: Sorry, I hate to be nitpicking here again, but can we please make sure that the sequence of paragraphs in the README is still consistent after we apply this change?

Currently, the paragraph above contains this:
```
 You can point your `aws` CLI to use the local infrastructure, for example:
```

... and the paragraph below your change contains this:
```
aws --endpoint-url=http://localhost:4568 kinesis list-streams
{
    "StreamNames": []
}
```
... (although these parts belong together and shouldn't be divided into separate subsections).

Overall, I would suggest something like this:
```
## Accessing the infrastructure via CLI or code

You can point your `aws` CLI to use the local infrastructure, for example:
---
aws --endpoint-url=http://localhost:4568 kinesis list-streams
{
    "StreamNames": []
}
---

Use the below command to install `aws CLI`, if not installed already.
---
pip install awscli
---

### Setting up local region and credentials to run requests against LocaLstack

aws requires the region and the credentials to be set in order to run the aws commands. Create the default configuration & the credentials. Below key will ask for the Access key id, secret Access Key, region & output format.
---
aws configure --profile default
# Config & credential file will be created under ~/.aws folder
---

**NEW**: Check out [awslocal](https://github.com/localstack/awscli-local), a thin CLI wrapper ...
```

Comment (rmsmani) [reply README.md:299]: :+1:

**Pull request merged**